### ISSUE TITLE
Image parameters maintained upon changing dataset

### ISSUE THREAD

**Issue by sjleake** [silx-kit/silx#2683]
Hi,
When loading an h5 file with many datasets, when inspecting a 2D slice and then changing to a another dataset to compare, if the datasets are the same dimension, it would be useful if the image or stackview maintain the chosen slice.
Thanks

**Comment by t20100** [comment]
PR #2701 provides part of it (i.e., it works for all views but the `StackView` which needs specific handling)

**Comment by Obi-Wan** [comment]
Any news on this? It seems that PR #2701 was merged, but this was not resolved. Viewing stacks of images is quite a useful application.

**Comment by t20100** [comment]
For image stacks, you can use the "Image" tab which keeps the selected slice when switching datasets.
The "Image stack" tab which is also available does not supports this.
Unless I missed something, the "Image stack" tab provides the same features as the "Image" one, so we'd better remove it (2 tabs providing the same feature is 1 too much).

**Comment by Obi-Wan** [comment]
I agree that the redundant tab should be removed as it can be confusing.
On the other hand, I have noticed that for 3D volumes, the labels of the axes differ a bit between the two tabs. In the image tab you have `{"", "x", "y"}` while in the stack tab `{"depth", "x", "y"}`. The empty label in the image tab can be confusing as well. So, before you do away with the stack tab, please give the dimension a label different from `""`. May I suggest `"z"`? :)
